Main.java

```diff
@@ -0,0 +1,8 @@
+public class Main {
+    public static void main(String[] args) {
+        Secondary secondary = new Secondary();
+        String value = secondary.printNumber();
+        System.out.println("~~~" + value + "~~~");

```

Comment:
This could be simplified to `System.out.println("~~~" + secondary.printNumber()+ "~~~");`, just suggestion